help/recent-conversations.md

```diff
@@ -9,6 +9,29 @@
     The arrow keys and vim navigation keys (<kbd>J</kbd>, <kbd>K</kbd>,
     <kbd>L</kbd>, <kbd>H</kbd>) can be used to move between elements.
 
+## Load more conversations
+
+The **recent conversations** view displays a limited number of conversations
+containing the most recent messages.
+
+!!! tip ""
+
+    Use the [search bar](/help/search-for-messages) at the top of the web app
+    to search through all conversations, not just recent ones.
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{!go-to-recent-conversations.md!}
+
+1. Scroll to the bottom of the list of conversations, or bring the bottom into
+   view by filtering for a non-existent keyword.
+
+2. Click the **Load more** button in the banner at the bottom.
```

Comment:
This is a nit, but we generally use "1." for all instructions.

 ```suggestion
1. Click the **Load more** button in the banner at the bottom.
```

Reply:
Heh, yeah, I should really figure out how to turn off my editor's auto-numbering.